apps/charterafrica/src/payload/collections/Organisations.js

```diff
@@ -0,0 +1,150 @@
+import { ORGANIZATION_COLLECTION } from "../../lib/tools/models";
+import slug from "../fields/slug";
+import source from "../fields/toolSourceField";
+import updatedAt from "../fields/updatedAt";
+
+const Organisations = {
+  slug: ORGANIZATION_COLLECTION,
+  admin: {
+    useAsTitle: "externalId",
+  },
+  access: {
+    read: () => true,
+  },
+  labels: {
+    singular: {
+      en: "Organisation",
+      fr: "Organisation",
+      pt: "Organização",
+    },
+    plural: {
+      en: "Organisations",
+      fr: "Organisations",
+      pt: "Organizações",
+    },
+  },
+  fields: [
+    source(),
+    {
+      name: "externalId",
+      type: "text",
+      label: { en: "External ID", fr: "ID externe", pt: "ID externo" },
+      required: true,
+      admin: {
+        readOnly: true,
+      },
+    },
+    {
+      name: "type",
+      type: "text",
+      label: {
+        en: "Type",
+        fr: "Type",
+        pt: "Tipo",
+      },
+      admin: {
+        readOnly: true,
+      },
+    },
+    {
+      name: "name",
+      type: "text",
+      label: {
+        en: "Organization Name",
```

Comment:
```suggestion
        en: "Name",
```

collection name is already Organization, no need to repeat it in field label.